mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -1567,6 +1577,118 @@ def _log_inputs_impl(
 
             session.add_all(objs_to_write)
 
+    def log_outputs(self, run_id: str, models: list[LoggedModelOutput]):
+        with self.ManagedSessionMaker() as session:
+            run = self._get_run(run_uuid=run_id, session=session)
+            self._check_run_is_active(run)
+            session.add_all(
+                SqlInput(
+                    input_uuid=uuid.uuid4().hex,
+                    source_type=OutputVertexType.RUN_OUTPUT,
+                    source_id=run_id,
+                    destination_type=OutputVertexType.MODEL_OUTPUT,
+                    destination_id=model.model_id,
+                    # step=model.step,
+                )
+                for model in models
+            )
```

Comment:
Should we use a separate table for outputs?

Reply:
FWIW we reuse the `inputs` table on Databricks. Works fairly well.

Reply:
@dbczumar Thanks for the comment. Is there a reason that we reused the inputs table?